source/fundamentals/indexes.txt

```diff
@@ -440,31 +440,17 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 Remove an Index
 ---------------
 
-You can remove any unused index except the default unique index on the
+You can delete any index from a collection except the default unique index on the
 ``_id`` field. To remove an index, pass the name of your index to the
 ``DropOne()`` method.
 
-The following example removes an ascending index on the ``title`` field
-in the ``sample_mflix.movies`` collection:
+The following example removes an index called ``title_1``
+from the ``sample_mflix.movies`` collection:
 
-.. io-code-block::
-   :copyable: true
-
-   .. input::
-      :language: go
-
-      coll := client.Database("sample_mflix").Collection("movies")
-      res, err := coll.Indexes().DropOne(context.TODO(), "title_1")
-      if err != nil {
-          panic(err)
-      }
-      fmt.Println(res)
-
-   .. output::
-      :language: none
-      :visible: false
+.. code-block:: go
 
-      {"nIndexesWas": {"$numberInt":"2"}}
+   coll := client.Database("sample_mflix").Collection("movies")
+   err := coll.Indexes().DropOne(context.TODO(), "title_1")
```

Comment:
Why do we remove the error-handling portion of this example?

Reply:
Oops, should have just removed the print statement, i will add the error handling back in!

Reply:
fixed!